wordle-web/scripts/wordService.ts

```diff
@@ -0,0 +1,41 @@
+import { LetterState } from "./letter";
+import { WordList } from "./wordList";
+
+export class WordService {
+  static validGuessedWords(guesses: string[], stateArray: LetterState[][]): string[] {
+    const validWords = new Set<string>();
+
+    for(let j = 0; j < guesses.length; j++){
+      const currentGuess: string = guesses[j];
+      const states: LetterState[] =  stateArray[j];
+
+      const isValidWord = WordList.filter(word => {
```

Comment:
`isValidWord` makes the variable sound like its a bool. Instead of creating a new list you might add to the validWords set like so:
```typescript
guesses.forEach((guess) => {
    guess = guess.toLowerCase();

    WordList.forEach((word) => {
      // if valid add to validWords
    });
  });
```

Reply:
Hey, thanks for the input on this.  I actually completely refactored how I did this part of the assignment since I found a better method.  Though this would have helped a lot if I used the same implementation. Cheers!